zcash_client_backend/src/data_api/chain.rs

```diff
@@ -44,7 +44,7 @@
 //! // Given that we assume the server always gives us correct-at-the-time blocks, any
 //! // errors are in the blocks we have previously cached or scanned.
 //! let max_height_hash = db_data.get_max_height_hash().map_err(Error::Wallet)?;
-//! if let Err(e) = validate_chain(&network, &block_source, max_height_hash) {
+//! if let Err(e) = validate_chain(&network, &block_source, max_height_hash, limit) {
```

Comment:
`limit` needs to be defined somewhere in the doctest source, or a constant provided like
```suggestion
//! if let Err(e) = validate_chain(&network, &block_source, max_height_hash, None) {
```